swablu/config.py

```diff
@@ -192,33 +192,18 @@ def vote_jam(dbcon, jam_key, user_id, hack):
 
 def update_hack(dbcon, hack, silent=False):
     cursor = db_cursor(dbcon)
-    if silent:
-        sql = f"UPDATE `{TABLE_NAME}` SET " \
-              f"`name` = %s," \
-              f"`description` = %s," \
-              f"`screenshot1` = %s," \
-              f"`screenshot2` = %s," \
-              f"`url_main` = %s," \
-              f"`url_discord` = %s," \
-              f"`url_download` = %s," \
-              f"`video` = %s," \
-              f"`hack_type` = %s," \
-              f"`message_id` = %s" \
-              f" WHERE id = %s"
-    else:
-        sql = f"UPDATE `{TABLE_NAME}` SET " \
-              f"`name` = %s," \
-              f"`description` = %s," \
-              f"`screenshot1` = %s," \
-              f"`screenshot2` = %s," \
-              f"`url_main` = %s," \
-              f"`url_discord` = %s," \
-              f"`url_download` = %s," \
-              f"`video` = %s," \
-              f"`hack_type` = %s," \
-              f"`message_id` = %s," \
-              f"`date_updated` = NOW()" \
-              f" WHERE id = %s"
+    sql = f"UPDATE `{TABLE_NAME}` SET " \
+          f"`name` = %s," \
+          f"`description` = %s," \
+          f"`screenshot1` = %s," \
+          f"`screenshot2` = %s," \
+          f"`url_main` = %s," \
+          f"`url_discord` = %s," \
+          f"`url_download` = %s," \
+          f"`video` = %s," \
+          f"`hack_type` = %s," \
+          f"`message_id` = %s" if silent else f"`message_id` = %s,`date_updated` = NOW()" \
```

Comment:
That works too. I would have done a variable, like this:
```py
date_updated_filter = ""
if not silent:
    date_updated_filter = ",`date_updated` = NOW()"
sql = f"... {date_updated_filter}"
```

Reply:
Oh, good to know. I'm not very familiar with how these strings work.